test/test.sh

```diff
@@ -69,7 +72,7 @@ function runtest() {
 
 
 	# change our target
-  cid=$(docker run --net mysqltest -d $DBDEBUG -e DB_USER=$MYSQLUSER -e DB_PASS=$MYSQLPW -e DB_DUMP_FREQ=60 -e DB_DUMP_BEGIN=+0 -e DB_DUMP_TARGET=${t2} -e AWS_ACCESS_KEY_ID=abcdefg -e AWS_SECRET_ACCESS_KEY=1234567 -e AWS_ENDPOINT_URL=http://s3:443/ -v /tmp/backups/${seqno}/post-backup:/scripts.d/post-backup -v /tmp/backups:/backups -e DBSERVER=mysql --link ${s3_cid}:mybucket.s3.amazonaws.com ${BACKUP_IMAGE})
+  cid=$(docker run --net mysqltest -d $DBDEBUG -e DB_USER=$MYSQLUSER -e DB_PASS=$MYSQLPW -e DB_DUMP_FREQ=60 -e DB_DUMP_BEGIN=+0 -e DB_DUMP_TARGET=${t2} -e DB_RESTORE_TARGET=${t2} -e AWS_ACCESS_KEY_ID=abcdefg -e AWS_SECRET_ACCESS_KEY=1234567 -e AWS_ENDPOINT_URL=http://s3:443/ -v /tmp/backups/${seqno}/:/scripts.d/ -v /tmp/backups:/backups -e DBSERVER=mysql --link ${s3_cid}:mybucket.s3.amazonaws.com ${BACKUP_IMAGE})
```

Comment:
The CI issue is here. You are running it with `DB_RESTORE_TARGET=${t2}`, but there is no db restore file provided.

Two possibilities:

1. Run it without `DB_RESTORE_TARGET`, which is fine, since we didn't have proper restore tests until now.
2. Insert db restore tests.

Although my instinct is to "do the right thing" and add db restore tests, we didn't have them until now, so I cannot complain. We should get this merged in, then deal with restore tests.

I think wrap this line with `[ -n "$TESTRESTORE" ]` as well, so you can run it with or without. Then we can add db restore tests in a separate PR.